extensions/ql-vscode/src/query-history.ts

```diff
@@ -812,6 +812,18 @@ export class QueryHistoryManager extends DisposableObject {
     await this.remoteQueriesManager.removeRemoteQuery(item.queryId);
   }
 
+  private async removeVariantAnalysis(item: VariantAnalysisHistoryItem): Promise<void> {
+    // We can remove a Variant Analysis locally, but not remotely.
+    // The user must cancel the query on GitHub Actions explicitly.
+    this.treeDataProvider.remove(item);
+    void logger.log(`Deleted ${this.labelProvider.getLabel(item)}.`);
+    if (item.status === QueryStatus.InProgress) {
+      void logger.log('The variant analysis is still running on GitHub Actions. To cancel there, you must go to the workflow run in your browser.');
```

Comment:
Should this be an informational notification rather than only a log message? We could then even provide a link to the Actions workflow.

Reply:
Thanks @koesie10! I've added a small issue for this.